### PR TITLE
RA-1591: Cosmetic tweaks for Manage Condition UI

### PR DESCRIPTION
**Pull request by haripriya999**
# RA-1591: Cosmetic tweaks for Manage Condition UI

## Description of what I changed
Made changes to UI

## Issue I worked on

see https://issues.openmrs.org/browse/RA-1591

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by HerbertYiga** [comment]
Did you do away with the HISTORY_OF radio button as i see in the snippet of your changes?

File: omod/src/main/webapp/pages/conditionlist/addCondition.gsp [hunk 1]
```diff
@@ -60,21 +60,21 @@
                         endDate    : new Date(),
                 ])}
             </li>
+            <div id="status" class="horizontal" style="margin-top:10px;">
```
Comment (dkayiwa) [inline omod/src/main/webapp/pages/conditionlist/addCondition.gsp]: Is this a formatting change? Are all these changes necessary? Can we just have only what you changed?

Comment (haripriya999) [reply omod/src/main/webapp/pages/conditionlist/addCondition.gsp]: Yes, the changes are necessary to get the alignment as the position for the element is relative.

**Comment by haripriya999** [comment]
> Did you do away with the HISTORY_OF radio button as i see in the snippet of your changes?

History of functionality has been deemed unnecessary according to: https://issues.openmrs.org/browse/RA-1461

**Comment by dkayiwa** [comment]
@haripriya999 can you look into these merge conflicts?

**Comment by haripriya999** [comment]
@dkayiwa  I resolved the merge conflict

**Comment by dkayiwa** [comment]
@haripriya999 did you see the travis build failure?

**Comment by haripriya999** [comment]
> @haripriya999 did you see the travis build failure?

@dkayiwa yah, It the same issue as reported in the talk - https://talk.openmrs.org/t/core-apps-travis-build-failure-the-job-exceeded-the-maximum-log-length-and-has-been-terminated/23774

**Pull request merged**